### ISSUE TITLE
ftplugin/python.vim: E518: Unknown option: ]\\?def\\|class\\)

### ISSUE THREAD

**Issue by skywind3000** [vim/vim#14774]
### Steps to reproduce

update to vim v9.1.0411:

edit `reproduce.vim`:

```viml
set nocompatible
filetype plugin on
```

start vim:

```bash
vim -u reproduce.vim
```

edit an arbitrary python file:

```bash
:edit foo.py
```

and you will see:

```text
Error detected while processing BufNewFile Autocommands for "*.py"..FileType Autocommands for "*"..function <SNR>25_LoadFTPlugin[18]..script /usr/local/opt/share/vim/vim
91/ftplugin/python.vim:
line   17:
E518: Unknown option: ]\\?def\\|class\\)
Press ENTER or type command to continue
```

screen capture:

![图片](https://github.com/vim/vim/assets/3035071/983c9d7f-8b5a-4b4c-80e5-96cc116dee83)

I believe it was introduced by https://github.com/vim/vim/commit/86f6e2c2eed7df2bf5c60cc74d08d7a8d3c75f45

### Expected behaviour

no exception.

### Version of Vim

9.1.411

### Environment

OS: Both Windows and Linux
Terminal: no matter terminal vim or gvim

**Comment by skywind3000** [comment]
@tpict , can you confirm this ?

**Comment by k-takata** [comment]
It has been already fixed in 7961210b861ce9a1a773cfc1fe96b388216bb1f2. Please try v9.1.0412.
Thanks.
